packages/storybook/stories/components/gallery.mdx

```diff
@@ -0,0 +1,315 @@
+import '../../assets/css/gallery.css';
+
+import { Canvas, Meta } from '@storybook/blocks';
+import { Overview as Accordion } from './accordion/accordion.stories';
+import { Default as Badge } from './badge/badge.stories';
+import { Default as Breadcrumb } from './breadcrumb/breadcrumb.stories';
```

Comment:
On the first render the breadcrumb don't render correctly
![image](https://github.com/user-attachments/assets/6ddd4064-0b32-4971-ba31-e373e166842a)